core/utils/http.go

```diff
@@ -6,13 +6,47 @@ import (
 	"fmt"
 	"io"
 	"io/ioutil"
+	"net"
 	"net/http"
 	"time"
 
 	"github.com/jpillora/backoff"
 	"github.com/smartcontractkit/chainlink/core/logger"
 )
 
+var (
+	Client             *http.Client
+	UnrestrictedClient *http.Client
+)
+
+func newDefaultTransport() *http.Transport {
+	// This is taken from the golang http client defaults
+	// See: https://golang.org/pkg/net/http/#Transport
+	t := http.Transport{
+		DialContext: (&net.Dialer{
+			Timeout:   30 * time.Second,
+			KeepAlive: 30 * time.Second,
+			DualStack: true,
+		}).DialContext,
+		ForceAttemptHTTP2:     true,
+		MaxIdleConns:          100,
+		IdleConnTimeout:       90 * time.Second,
+		TLSHandshakeTimeout:   10 * time.Second,
+		ExpectContinueTimeout: 1 * time.Second,
+	}
+	t.DisableCompression = true
```

Comment:
hm why not just inside the literal def?

Reply:
Because the top part is copied verbatim from the golang docs.

DisableCompression is our addition